hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/streams/assertions/ValidContractIdsAssertion.java

```diff
@@ -68,6 +69,10 @@ private void validateActionIds(@NonNull final TransactionSidecarRecord sidecar)
                 assertValid(action.getRecipientContract(), "action#recipientContract", sidecar, this::isValidId);
             }
 
+            final var isMissingRecipient =
```

Comment:
(It's just me, I'm sure, but I would have use De Morgan's law to make this be `hasARecipient` and done everything in the _positive_ sense.  Because I'm a positive kind of guy.   Next line too.  But don't change anything here.  Just a thought on people's differing world-views and how that's reflected in code.  Or, possibly, that's just baloney, huh?)

Reply:
Switched to the positive spin. 😁